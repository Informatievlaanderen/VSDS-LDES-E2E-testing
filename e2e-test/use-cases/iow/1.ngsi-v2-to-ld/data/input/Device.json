{
    "id": "urn:ngsi-v2:cot-imec-be:Device:dwg-iow-T83eGVGxSyo5srYHN2dCxU",
    "type": "Device",
    "refDeviceModel": {
        "type": "Relationship",
        "value": "urn:ngsi-v2:cot-imec-be:devicemodel:ysi-exo2",
        "metadata": {}
    },
    "alternativeIds": {
        "type": "StructuredValue",
        "value": [
            "urn:ngsi-v2:cot-imec-be:WaterQualityObserved:dwg-iow-6LTvUm38Hr4uR3MoapAAFZ"
        ],
        "metadata": {}
    },
    "maxTimeBetweenObservations": {
        "type": "Number",
        "value": 420,
        "metadata": {}
    },
    "category": {
        "type": "StructuredValue",
        "value": [
            "sensor"
        ],
        "metadata": {}
    },
    "locationName": {
        "type": "Text",
        "value": "loc-00052-31",
        "metadata": {}
    },
    "controlledProperty": {
        "type": "StructuredValue",
        "value": [
            "temperature",
            "conductivity",
            "pH",
            "O2",
            "turbidity"
        ],
        "metadata": {}
    },
    "source": {
        "type": "String",
        "value": "cot.iow.observed",
        "metadata": {}
    },
    "healthState": {
        "type": "Text",
        "value": "up",
        "metadata": {}
    },
    "location": {
        "type": "geo:json",
        "value": {
            "type": "Point",
            "coordinates": [
                2.8436689,
                50.9923557
            ]
        },
        "metadata": {
            "name": {
                "type": "Text",
                "value": "loc-00052-31"
            }
        }
    },
    "adminState": {
        "type": "Text",
        "value": "up",
        "metadata": {}
    },
    "extraMeasurementAttributes": {
        "type": "StructuredValue",
        "value": {
            "location": true,
            "samplingPoint": true,
            "locationName": true
        },
        "metadata": {}
    },
    "dateLastValueReported": {
        "type": "DateTime",
        "value": "2022-01-24T20:00Z",
        "metadata": {}
    },
    "name": {
        "type": "Text",
        "value": "dwg-box-exo-Blankaart",
        "metadata": {}
    },
    "samplingPoint": {
        "type": "Text",
        "value": "spt-00097-87",
        "metadata": {}
    }
}