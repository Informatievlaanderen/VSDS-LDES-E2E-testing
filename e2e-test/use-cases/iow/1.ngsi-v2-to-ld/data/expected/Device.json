{
    "@context": [
        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"
    ],
    "id": "urn:ngsi-v2:cot-imec-be:Device:imec-iow-UR5gEycRuaafxnhvjd9jnU",
    "type": "Device",
    "adminState": {
        "type": "Property",
        "value": "down"
    },
    "alternativeIds": {
        "type": "Property",
        "value": [
            "simcard.206012902136977",
            "octa_dec.2336861766884020237"
        ]
    },
    "batteryLevel": {
        "type": "Property",
        "value": 0.32,
        "rawValue": {
            "type": "Property",
            "value": "3192"
        },
        "rawValueUnitCode": {
            "type": "Property",
            "value": "2Z"
        }
    },
    "calibrationParameters": {
        "type": "Property",
        "value": {
            "battery": {
                "gain": 600,
                "offset": 3000
            }
        }
    },
    "copyObservationAttributes": {
        "type": "Property",
        "value": [
            "batteryLevel"
        ]
    },
    "dateLastValueReported": {
        "type": "Property",
        "value": {
            "@type": "DateTime",
            "@value": "2022-06-28T08:26:06.000Z"
        }
    },
    "extraMeasurementAttributes": {
        "type": "Property",
        "value": {
            "location": true,
            "locationName": true,
            "samplingPoint": true
        }
    },
    "healthState": {
        "type": "Property",
        "value": "up"
    },
    "location": {
        "type": "GeoProperty",
        "value": {
            "type": "Point",
            "coordinates": [
                5.4563,
                51.41363
            ]
        },
        "name": {
            "type": "Property",
            "value": "loc-00056-19"
        }
    },
    "locationName": {
        "type": "Property",
        "value": "loc-00056-19"
    },
    "maxTimeBetweenObservations": {
        "type": "Property",
        "value": 60
    },
    "name": {
        "type": "Property",
        "value": "box-00145-49"
    },
    "owner": {
        "type": "Property",
        "value": [
            "imec"
        ]
    },
    "refDeviceModel": {
        "type": "Relationship",
        "object": "urn:ngsi-v2:cot-imec-be:devicemodel:imec-iow-box-v0003"
    },
    "samplingPoint": {
        "type": "Property",
        "value": "spt-00000-87"
    },
    "source": {
        "type": "Property",
        "value": "cot.iow.observed"
    }
}